concordium-consensus/src/Concordium/GRPC2.hs

```diff
@@ -546,29 +826,120 @@ enqueueMessages callback = forkIO . go 0 . map encodeMsg
+
+foreign export ccall
+    getAccountListV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        Ptr SenderChannel ->
+        -- |Block type.
+        Word8 ->
+        -- |Block hash.
+        Ptr Word8 ->
+        -- |Out pointer for writing the block hash that was used.
+        Ptr Word8 ->
+        FunPtr (Ptr SenderChannel -> Ptr Word8 -> Int64 -> IO Int32) ->
+        IO Int64
+
+foreign export ccall
+    getModuleListV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        Ptr SenderChannel ->
         -- |Block type.
         Word8 ->
         -- |Block hash.
         Ptr Word8 ->
         -- |Out pointer for writing the block hash that was used.
         Ptr Word8 ->
+        FunPtr (Ptr SenderChannel -> Ptr Word8 -> Int64 -> IO Int32) ->
+        IO Int64
+
+foreign export ccall
+    getModuleSourceV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        -- |Block type.
+        Word8 ->
+        -- |Block hash.
+        Ptr Word8 ->
+        -- |Module reference.
+        Ptr Word8 ->
+        -- |Out pointer for writing the block hash that was used.
+        Ptr Word8 ->
         Ptr ReceiverVec ->
         FunPtr (Ptr ReceiverVec -> Ptr Word8 -> Int64 -> IO ()) ->
         IO Int64
 
 foreign export ccall
-    getAccountListV2 ::
+    getInstanceListV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        Ptr SenderChannel ->
+        -- |Block type.
+        Word8 ->
+        -- |Block hash.
+        Ptr Word8 ->
+        -- |Out pointer for writing the block hash that was used.
+        Ptr Word8 ->
+        FunPtr (Ptr SenderChannel -> Ptr Word8 -> Int64 -> IO Int32) ->
+        IO Int64
+
+foreign export ccall
+    getAncestorsV2 ::
         StablePtr Ext.ConsensusRunner ->
         Ptr SenderChannel ->
         -- |Block type.
         Word8 ->
         -- |Block hash.
         Ptr Word8 ->
+        -- |Amount of ancestors (depth).
+        Word64 ->
         -- |Out pointer for writing the block hash that was used.
         Ptr Word8 ->
         FunPtr (Ptr SenderChannel -> Ptr Word8 -> Int64 -> IO Int32) ->
         IO Int64
+
+foreign export ccall
+    getInstanceInfoV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        -- |Block type.
+        Word8 ->
+        -- |Block hash.
+        Ptr Word8 ->
+        -- |Smart contract address index.
+        Word64 ->
+        -- |Smart contract address subindex.
+        Word64 ->
+        -- |Out pointer for writing the block hash that was used.
+        Ptr Word8 ->
+        Ptr ReceiverVec ->
+        FunPtr (Ptr ReceiverVec -> Ptr Word8 -> Int64 -> IO ()) ->
+        IO Int64
+
+foreign export ccall
+    getNextAccountSequenceNumberV2 ::
+        StablePtr Ext.ConsensusRunner ->
+        -- |Identifier type, 0 for account address, 1 for credential, 2 for account index
+        Word8 ->
+        -- |Serialized identifier. Length determined by the type.
+        Ptr Word8 ->
+        Ptr ReceiverVec ->
+        FunPtr (Ptr ReceiverVec -> Ptr Word8 -> Int64 -> IO ()) ->
```

Comment:
You also need to add these new functions to the `lib.def`  file. Otherwise Windows people will dislike you :)